app/services/post_status_service.rb

```diff
@@ -40,8 +40,7 @@ def call(account, text, in_reply_to = nil, **options)
     LinkCrawlWorker.perform_async(status.id) unless status.spoiler_text?
     DistributionWorker.perform_async(status.id)
 
-    # match both with and without U+FE0F (the emoji variation selector)
-    unless /👁\ufe0f?\z/.match?(status.content)
+    unless status.local_only
```

Comment:
not really a big deal, but this would more typically be `status.local_only?`

Reply:
It was originally that, bea suggested that instead. I'll switch it back, since it's exactly the same thing (thanks, ActiveRecord :p)